testing/src/main/java/org/conscrypt/java/security/StandardNames.java

```diff
@@ -163,6 +163,9 @@ private static void provideSslContextEnabledProtocols(
             Arrays.asList(SSL_CONTEXT_PROTOCOLS_DEFAULT, "TLS", "TLSv1", "TLSv1.1", "TLSv1.2", "TLSv1.3"));
     public static final Set<String> SSL_CONTEXT_PROTOCOLS_WITH_DEFAULT_CONFIG = new HashSet<String>(
             Arrays.asList(SSL_CONTEXT_PROTOCOLS_DEFAULT, "TLS", "TLSv1.3"));
+    // Deprecated TLS protocols... May or may not be present or enabled.
+    public static final Set<String> SSL_CONTEXT_PROTOCOLS_DEPRECATED = new HashSet<>(
```

Comment:
nit: SSL_CONTEXT_PROTOCOLS_ENABLED is Map, this one is Set. Can't come up with better naming though.

Reply:
In a similar C: I was working on (which adds an API for disabling TLSv1), I suffixed it with `_SET`, but I don't feel strongly.

There's a lot that could be tidied up in here, especially the mish-mash of arrays vs collection classes (contributions welcome ;) but that I wanted to keep the changes fairly minimal in this CL to ease backporting.